addon/converter/fixture-converter.js

```diff
@@ -39,7 +39,8 @@ export default class {
     if ('function' === typeof serializer.transformFor) {
       // Use serializer.transformFor for compatibility with ember-data-model-fragments,
       // for context see https://github.com/danielspaniel/ember-data-factory-guy/issues/182
-      return serializer.transformFor(type).serialize;
+      let fragmentType = serializer.transformFor(type);
+      return fragmentType.deserialize.bind(fragmentType);
```

Comment:
Right, that is what I was saying about the (x)=> x .. function = noTransformFn. 
 I think that is the correct answer .. but I want to get more tests in there and if you can help with that .. then rock on. 
  Need to know more about MF transformations and when they are used.